cmd/juju/commands/bootstrap.go

```diff
@@ -345,6 +347,9 @@ func (c *bootstrapCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.StringVar(&c.ControllerCharmChannelStr, "controller-charm-channel",
 		fmt.Sprintf("%d.%d/stable", jujuversion.Current.Major, jujuversion.Current.Minor),
 		"The Charmhub channel to download the controller charm from (if not using a local charm)")
+	if jujuversion.Current.Build > 0 {
+		f.BoolVar(&c.Dev, "dev", false, "Use local build for development")
+	}
```

Comment:
This is annoying if you do `-h` or `--help`! It will only show if the build is greater than 0.

Reply:
I thought it was a good thing. We don't want "--dev" to be used for builds minted by jenkins, only builds minted locally. Sure, I can set JUJU_BUILD_NUMBER=0 locally, but if I am doing that, then I was probably wanting to test "released" behavior.

Reply:
Leave it for now, we can iterate on it if required.